ibis/backends/pandas/execution/generic.py

```diff
@@ -920,6 +920,7 @@ def execute_database_table_client(
     df = client.dictionary[op.name]
     if timecontext:
         begin, end = timecontext
+        TIME_COL = get_time_col()
```

Comment:
This looks funky, why are we setting a constant here?

Reply:
ok yeah, upper case is confusing because it looks like you are reassigning value to a constant.